### PR TITLE
Update sdg.yaml

### PR DESCRIPTION
**Pull request by VassilisDaioglou**
Added Population| 

- Sanitation Access
- Drinking Water Access

### CODE REVIEW

File: definitions/variable/sdg-indicators/sdg.yaml [hunk 1]
```diff
@@ -31,6 +31,14 @@
       gas)
     unit: million
     navigate: Population|Clean cooking Access
+- Population|Sanitation Access:
+    description: Population with access to improved sanitation
+    unit: million
+    shape: Population|Sanitation Access
```
Comment (danielhuppmann) [inline definitions/variable/sdg-indicators/sdg.yaml:37]: Reference to "shape" (or other projects) can be omitted if the variable name did not change. This is only intended for automated mapping across projects.
```suggestion
```

File: definitions/variable/sdg-indicators/sdg.yaml [hunk 2]
```diff
@@ -31,6 +31,14 @@
...
+- Population|Drinking Water Access:
+    description: Population with access to improved drinking  water sources 
+    unit: million
+    shape: Population|Drinking Water Access
```
Comment (danielhuppmann) [inline definitions/variable/sdg-indicators/sdg.yaml:41]: Can be removed as above.
```suggestion
```

**Review by danielhuppmann — COMMENTED** [review]
Two comments inline. I'm wondering I we shouldn't move these variables "access to ..." to an own (sub)-category.

**Comment by VassilisDaioglou** [comment]
RE moving to a different sub-category could be something to be considered. eIn the future a number of "Access"variables will beome important. 

They all fall under "Population|Access|[...]", so it could also fall under the "Population" sub-category with a "tag_access".

File: definitions/variable/sdg-indicators/sdg.yaml [hunk 3]
```diff
@@ -31,6 +31,12 @@
       gas)
     unit: million
     navigate: Population|Clean cooking Access
+- Population|Sanitation Access:
+    description: Population with access to improved sanitation
+    unit: million
```
Comment (danielhuppmann) [inline definitions/variable/sdg-indicators/sdg.yaml:36]: Sorry, just realized that this variable actually did exist, just misspelled..
```suggestion
    unit: million
    navigate: Population|Sanitation Acces
    engage: Population|Sanitation Acces
```

**Comment by danielhuppmann** [comment]
Well, I thought maybe moving this to a new top-level category like "Decent Living Indicator" - we already have too many items categorized under "Population". But not sure if that's a good move right now...

**Comment by danielhuppmann** [comment]
@VassilisDaioglou, can you commit my proposed change to add the mapping to the mis-spelt variables in earlier projects, then I'll merge.

**Comment by VassilisDaioglou** [comment]
Done - sorry for the delay.

**Review by danielhuppmann — APPROVED** [review]
Thanks, looks good!

**Pull request merged**